code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1108,20 +1108,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	id = "hearty_punch"
 	description = "Brave bull/syndicate bomb/absinthe mixture resulting in an energizing beverage. Mild alcohol content."
 	color = rgb(140, 0, 0)
-	boozepwr = 10
-	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+	boozepwr = 90
+	metabolization_rate = 1.0 * REAGENTS_METABOLISM
```

Comment:
1*x = x
You can just remove multiplying it by 1.0 and it'll be the same calculation as with it.

Reply:
if you keep metab as is and tenfold raise the boozepwr that should heck over your liver for sure no?